app/helpers/shop_helper.rb

```diff
@@ -20,27 +20,24 @@ def user_is_related_to_distributor?
     )
   end
 
-  def base_shop_tabs(column_sizes)
+  def shop_tabs
     [
-      { name: 'about', cols: column_sizes[0],
-        title: t(:shopping_tabs_about, distributor: current_distributor.name) },
-      { name: 'producers', cols: column_sizes[1],
-        title: t(:label_producers) },
-      { name: 'contact', cols: column_sizes[2],
-        title: t(:shopping_tabs_contact) }
+      { name: 'home', title: t(:shopping_tabs_home), show: show_home_tab? },
+      { name: 'shop', title: t(:shopping_tabs_shop), show: !require_customer? },
+      { name: 'about', title: t(:shopping_tabs_about), show: true },
+      { name: 'producers', title: t(:label_producers), show: true },
+      { name: 'contact', title: t(:shopping_tabs_contact), show: true },
+      { name: 'groups', title: t(:label_groups), show: current_distributor.groups.any? },
     ]
   end
 
-  def tabs_with_groups
-    tabs = base_shop_tabs([6, 2, 2])
-    tabs << { name: 'groups', title: t(:label_groups), cols: 2 }
-  end
+  private
```

Comment:
This `private` is declaring an intent only. The methods below can still be accessed from views. Helpers are modules that are included in the view.

Another developer might think that this `private` is actually working. What do you think about replacing it with a comment? Or move this logic to a presenter or service?